src/util/bit_util.rs

```diff
@@ -561,20 +564,6 @@ mod tests {
   }
 
   #[test]
-  #[should_panic(expected = "Num of bytes exceed MAX_VLQ_BYTE_LEN (5)")]
-  fn test_bit_reader_get_vlq_int_overflow() {
```

Comment:
I removed this test for one reason - could not find example of byte sequence that will panic for MAX_VLQ_BYTE_LEN(10). IMHO, I am not sure if we need this test at all now, since I increased to 10, and it covers all i64 values.